### ISSUE TITLE
Release flagged as malicious

### ISSUE THREAD

**Issue by hlaueriksson** [Fefedu973/PowerToys-Run-Plugin-Updater#1]
Hi @Fefedu973

I ran the latest release through VirusTotal and got this result:

- https://www.virustotal.com/gui/file/ff841612f6559f53c6748ac8cba6861d4841dfa1123915b2c0dc2561a3ab15e9/detection

**Comment by Fefedu973** [comment]
Bruh, I think this is because the app is downloading the plugin in the background when it finds an update which is a common thing that viruses do

**Comment by Fefedu973** [comment]
Could you please try to build it yourself and compare the hash and put it in virus total to check if there is the same detection?

**Comment by Fefedu973** [comment]
?

**Comment by hlaueriksson** [comment]
Yes, I also got the `win-x64` exe I published flagged:

- https://www.virustotal.com/gui/file/947e5f6bac98346370534a97167fa43b505e11b5cf361c8985d76eaac21bb3ab

**Comment by Fefedu973** [comment]
> Yes, I also got the `win-x64` exe I published flagged:
> 
> - https://www.virustotal.com/gui/file/947e5f6bac98346370534a97167fa43b505e11b5cf361c8985d76eaac21bb3ab

Ok so that means that if you check the code (it's open source) and you find nothing malicious (you won't find anything malicious because it is not) that this software is not a malware...
Can I close the issue or not ?
Thanks for your Involvement it saves me of the same question but from people that don't know how all of that works...

**Comment by hlaueriksson** [comment]
Yes, I understand that this is no malicious code in the repo. However, I would recommend to try figure out why some antivirus providers flag this code. It would build trust among users if the release was clean.
